pkg/jx/cmd/add_app.go

```diff
@@ -74,15 +74,15 @@ func NewCmdAddApp(f Factory, in terminal.FileReader, out terminal.FileWriter, er
 		},
 	}
 
-	options.addFlags(cmd, kube.DefaultNamespace, "", "")
+	options.addFlags(cmd, kube.DefaultNamespace)
```

Comment:
> package cmd

Didn't you say that you started to have split up the _cmd_ package? Would it make sense to have this in a _add_ subdirectory? Or is the idea to restructure this in a big bang effort?

Reply:
Yeah, we could do. I'll do that as a separate PR though!